node/config.go

```diff
@@ -190,6 +194,9 @@ type Config struct {
 
 	// AllowUnprotectedTxs allows non EIP-155 protected transactions to be send over RPC.
 	AllowUnprotectedTxs bool `toml:",omitempty"`
+
+	// JwtSecret is the hex-encoded jwt secret.
+	JwtSecret string `toml:",omitempty"`
```

Comment:
Nit, iw we used uppercase for `HTTP` and `WS` ad `IPC` and `USB`, we shouls also use full uppercase for `JWT`